bay/docker/introspect.py

```diff
@@ -45,15 +48,14 @@ def add_container(self, container_details):
         instance = self._create_container(container_details)
         self.formation.add_instance(instance)
 
-    def _create_container(self, container_details):
+    def _create_container(self, container_name):
         """
         Returns a container build from introspected information
         """
-        # Get the name
-        container_name = container_details['Names'][0].lstrip("/")
+        container_details = self.host.client.inspect_container(container_name)
         # Find the container name in the graph
         try:
-            labels = container_details['Labels']
+            labels = container_details['Config']['Labels']
             container = self.graph[labels['com.eventbrite.bay.container']]
```

Comment:
Let's document what this label means in terms of container details, since `eventbrite` is in there.